Userland/Libraries/LibJS/Runtime/IteratorOperations.cpp

```diff
@@ -88,7 +96,9 @@ ThrowCompletionOr<Object*> iterator_step(GlobalObject& global_object, Object& it
 }
 
 // 7.4.6 IteratorClose ( iteratorRecord, completion ), https://tc39.es/ecma262/#sec-iteratorclose
-Completion iterator_close(Object& iterator, Completion completion)
+// 7.4.8 AsyncIteratorClose ( iteratorRecord, completion ), https://tc39.es/ecma262/#sec-asynciteratorclose
+// NOTE: These only differ in that async awaits the inner value after the call.
+static Completion iterator_close_(Object& iterator, Completion completion, IteratorHint is_async)
```

Comment:
`is_async` is the name of a boolean, please call it something else (iterator_hint, for example)